### DISCUSSION TITLE
Create Rest Services within Solara

### DISCUSSION THREAD

**Discussion by icduck** [widgetti/solara#301] [General]
Hi all,

Have anyone created the API rest services within Solara app?
I looked around but have not been able to find out anything yet. Any hint would be appreciated.
Thank you

**Comment by maartenbreddels** [comment]
Hi 🦆,

Have you seen https://solara.dev/docs/deploying/self-hosted ? Does this answer your question?
Or do you want to use the `solara run ..` command line util and add an extra endpoint?

Regards,

Maarten

**Reply by icduck** [reply]
Thank you, Maarten! it works.
Beautiful Solara! more than what I expected.

**Reply by maartenbreddels** [reply]
Thank you 🙏
